packages/vuetify/src/composables/proxiedModel.ts

```diff
@@ -28,7 +28,7 @@ export function useProxiedModel<
     )
   })
 
-  const internal = ref(transformIn(propIsDefined.value ? props[prop] : defaultValue)) as Ref<Inner>
+  const internal = ref(transformIn(props[prop])) as Ref<Inner>
```

Comment:
What was this change for? It breaks the snapshots.

Reply:
might be experiment I forgot to revert